README.md

```diff
@@ -151,3 +151,83 @@ operation not supported
 ```
 This is probably because of a docker bug where, if you update kernel and don't reboot, docker gets confused. The fix is to reboot your system.
 More info: http://stackoverflow.com/questions/29546388/getting-an-operation-not-supported-error-when-trying-to-run-something-while-bu
+
```

Comment:
@willnewton I guess the NTP and DNS stuff will get added to the list when they are merged right?